kernel/configs/fragments/common/base.conf

```diff
@@ -48,4 +48,4 @@ CONFIG_RANDOMIZE_BASE=y
 # This can still be dynamically disabled on the kernel command line/kata config if needed.
 # Disable for now, as it upsets the entropy test, and we need to improve those: FIXME: see:
 # https://github.com/kata-containers/tests/issues/1543
-# CONFIG_RANDOM_TRUST_CPU is not set
```

Comment:
I'm not sure that is/was a typo - iirc enabling it broke entropy - we need to review this one I think.